### PR TITLE
Update delete account

### PR DESCRIPTION
**Pull request by SirSaltyy**
-Now deletes *all personal info* from firebase.
-Improved copy to be more transparent.

### CODE REVIEW

**Comment by SirSaltyy** [comment]
FYI @mantikoros I wasn't able to test the backend functionality, as my dev.sh environment is still not working.

As far as I can tell I don't need to redeploy anything once merged for these changes to come into effect as I'm not changing any backend functions, but let me know if that isn't the case. When I try to delete in prod local host I get a "failed to delete account" though, not sure if it's due to changes I made or just because it's in the wrong environment.

**Review by mantikoros — APPROVED** [review]
LGTM, but don't merge this in until after you've tested this on dev.

File: web/pages/profile.tsx
Comment (IanPhilips) [inline web/pages/profile.tsx:118]: I'm not sure I think this is a good idea. What about trolls that come onto the site, wreak havoc with spam, then go and delete their account - how can we ban their ip address going forward?

File: web/pages/profile.tsx
Comment (IanPhilips) [inline web/pages/profile.tsx:121]: This isn't really personalized, just basically a cookie on their browser

**Comment by IanPhilips** [comment]
I think we should hold off on this until we have a good solution for handling banned users' ip addresses.

**Pull request closed without merging**
